Platform/CoffeelakeBoardPkg/CfgData/Template_CfgData.dsc

```diff
@@ -227,3 +227,14 @@
   # ---------------------------------------------------------------------------------------
   # RP_TMPL
   !include Template_PcieRp.dsc
+
+  # ---------------------------------------------------------------------------------------
+  # SPD_TMPL
+  !include Template_Spd.dsc
+
+  # ---------------------------------------------------------------------------------------
+  # MEM_TMPL
```

Comment:
MEM_TMPL  is only used once,  why do we need a template here ?

Reply:
Removed MEM_TMPL